esm/interface/event-target.js

```diff
@@ -5,4 +5,22 @@ import EventTarget from '@ungap/event-target';
 /**
  * @implements globalThis.EventTarget
  */
-export {EventTarget};
+class DOMEventTarget extends EventTarget {
+  getParent() {
```

Comment:
I'd rather go for `_getParent()` because having a "*global method*" might be future hostile.